bindings/nodejs/lib/index.js

```diff
@@ -1,143 +1,52 @@
-// Copyright 2020 IOTA Stiftung
+// Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-const addon = require('../build/Release')
-let { AccountManager, Account, EventListener, initLogger } = addon
+const addon = require('../index.node');
+// const mh = require("./messages/messageHandler.js");
```

Comment:
Can we export both things at the same time so one doesn't need to uncomment one?